framework/packages/abstract-sdk/Cargo.toml

```diff
@@ -13,6 +13,7 @@ resolver = "2"
 [features]
 default = []
 stargate = ["dep:cosmos-sdk-proto", "dep:prost-types"]
+tokenfactory = ["stargate", "dep:osmosis-std", "dep:osmo-bindings"]
```

Comment:
this uses the osmosis bindings but as mentioned [here](https://twitter.com/larry0x/status/1641096055310757890) there are a ton of different "standards". 

Not sure if we want to commit to any of them in this way.

Reply:
Yes - found this out recently as well. What do you feel would be the best course of action? Separate features for each? `osmosis-tokenfactory`?